docs/RESOURCES.md

```diff
@@ -1,20 +1,19 @@
 # Resources
 
-To get help if you're having trouble, you can use one of the following resources:
+If you require assistance with anything concerning Ballerina, utilize the resources provided below.
 
-- the Ballerina web site
-    - [Getting Started][getting-started]
-    - [Ballerina Examples][ballerina-examples]
-    - [Join Discord][join-discord]
-- [StackOverflow][stackoverflow]
-- Exercism's resources
-    - get help on [the forum][exercism-forum], or
-    - in the #support channel on [Exercism's discord][exercism-discord].
+- Ballerina official website
```

Comment:
Instead of official website, say something like "Ballerina learning resources"

Reply:
Changed.